bin/entrypoint

```diff
@@ -33,4 +33,4 @@ if ${OC_LOGIN:-true} ; then
   oc_login
 fi
 
-exec ${@}
+exec "${@}"
```

Comment:
That's funny because we had to take out these double quotes to make the image work on some Macs... We will need to do some follow-up if we merge this.

Reply:
Yes we must to test this on Macs.